vignettes/summary_selecting.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "A quick summary of selecting possibilities in `datawizard`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{A quick summary of selecting possibilities in `datawizard`}
```

Comment:
```suggestion
  %\VignetteIndexEntry{A quick summary of selecting syntax in `{datawizard}`}
```

I hope this can work. If not, we can use just `{datawizard}`, and not `` `{datawizard}` ``